scaleway/helpers_iot.go

```diff
@@ -0,0 +1,50 @@
+package scaleway
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	iot "github.com/scaleway/scaleway-sdk-go/api/iot/v1beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	IotHubWaitTimeout = 10 * time.Minute
```

Comment:
Should be in the SDK https://github.com/scaleway/scaleway-sdk-go/blob/master/api/lb/v1/lb_utils.go#L11